### PR TITLE
Modifying ReutersBridge

### PR DESCRIPTION
**Pull request by csisoap**
- Add UID for each article.
- Add images, full details of article and author's name.
- Refactor some code.

P/s: The feed may takes time to instantiate or when the cache refreshed and because for every article, it need to get the full detail of that.

image

### CODE REVIEW

**Comment by csisoap** [comment]
@hollowleviathan

**Comment by hollowleviathan** [comment]
Is adding the default Reuters logo image to image-less news stories better than just not having images in those stories?

File: bridges/ReutersBridge.php [hunk 1]
```diff
@@ -1,101 +1,198 @@
...
+            $author = $content_detail['author'];
+            $item['author'] = $author;
+            if (!(bool) $description) {
+                $description = $story['story']['lede']; // Just in case the content doesn't have anything.
+            }
+            #	$description = $story['story']['lede'];
+            $image_url = $story['image']['url'];
+            if (!(bool) $image_url) {
+                $image_url =
+                    'https://s4.reutersmedia.net/resources_v2/images/rcom-default.png'; //Just in case if there aren't any pictures.
```
Comment (hollowleviathan) [inline bridges/ReutersBridge.php]: Is adding the default Reuters logo image to image-less news stories better than just not having images in those stories?

Comment (csisoap) [reply bridges/ReutersBridge.php]: Yeah, it's better than not to have that.

Comment (csisoap) [reply bridges/ReutersBridge.php]: Fixed that in the latest commit.

**Comment by hollowleviathan** [comment]
Could you please run `phpcbf` on your  bridges/ReutersBridge.php, use tabs instead of spaces, and use `array()` instead of `[]` to conform with the RSS-Bridge style guide?

I've enabled the Travis CI that auto-runs on the MR and points out what changes RSS-Bridge wants

**Comment by csisoap** [comment]
I've ran PHPCBF on the code, change all `[]` into `array()` and it still fail. The code passed the syntax test but not indentation/whitespace test.

UPDATE: The code has passed all test. I've tabs all the code and then PHPCBF change back to spaces, i don't know why?

**Pull request merged**